CO2_Cleanup.ipynb

```diff
@@ -0,0 +1,149 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #75.            cleaned_series = pd.Series(processed_data_dict[continent], index=raw_data.index)
</pre><p>this conclicts with your dropna processing from earlier</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/CP1-24-HW5/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>